cmd/habitat/community/state/transitions.go

```diff
@@ -80,16 +75,30 @@ func (t *IncrementCounterTransition) Patch(oldState *community.CommunityState) (
 	}]`, oldState.Counter+1)), nil
 }
 
-func (t *IncrementCounterTransition) JSON(oldState *community.CommunityState) ([]byte, error) {
-	patch, err := t.Patch(oldState)
-	if err != nil {
-		return nil, err
-	}
+func (t *IncrementCounterTransition) Validate(oldState *community.CommunityState) error {
+	return nil
+}
 
-	return serializeTransition(patch, TransitionTypeIncrementCounter)
+type InitializeCommunityTransition struct {
+	CommunityID string
 }
 
-func (t *IncrementCounterTransition) Validate(oldState *community.CommunityState) error {
+func (t *InitializeCommunityTransition) Type() string {
+	return TransitionTypeInitializeCommunity
+}
+
+func (t *InitializeCommunityTransition) Patch(oldState *community.CommunityState) ([]byte, error) {
```

Comment:
ooc, why is this called patch?

Reply:
It applies a JSON patch to the existing JSON data structure that represents state